content/en/agent/docker/log.md

```diff
@@ -57,6 +57,17 @@ docker run -d --name datadog-agent \
            datadog/agent:latest
 ```
 
+**Note**: On Windows systems, run this command without volume mounts. That is:
+
+```shell
+docker run -d --name datadog-agent \
+           -e DD_API_KEY="<DATADOG_API_KEY>" \
+           -e DD_LOGS_ENABLED=true \
+           -e DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL=true \
+           -e DD_AC_EXCLUDE="name:datadog-agent" \
```

Comment:
You'd need to add `-v \\.\pipe\docker_engine:\\.\pipe\docker_engine`

Reply:
It needs to be before the the image name `datadog/agent:latest` otherwise it'll be considered as arguments to the entrypoint.